tailscale/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "e3f4a5cf-3594-43fc-9d4e-4e86b9c91ea2",
+  "app_id": "tailscale",
+  "display_on_public_website": false,
```

Comment:
Do you want the docs to be displayed on our public docs?

Reply:
Would I be able to put it behind a feature flag for just 2 orgs?

Reply:
Heya! Don't mind to just interject into the conversation here- I'm Day from our Detection Engineering team supporting the development of the detection rules, logs pipeline and dashboard for this integration. 

@FlorentClarret Would it make sense to also add the logs-pipeline and dashboard code within the assets folder for testing alongside the tile in staging or would that need to be done from a separate branch after this is merged?

Reply:
> Would I be able to put it behind a feature flag for just 2 orgs?

We have two different things:

1. The public documentaton (for instance [this](https://docs.datadoghq.com/integrations/apache/?tab=host) for Apache) is published using the `display_on_public_website` flag. There's no way to put it behing a feature flag since it's a public documentation. My question is: do you want to publish the docs as soon as we publish the tile or do you want us to wait for a specific date? (announcement, event, etc...)
2. The tile itself that is showed in-app (for instance [this](https://app.datadoghq.com/integrations/apache) for Apache). This one can be put behind a feature flag if you want to have the tile on prod only for some orgs. If that's what you'd like, please reach out to me on Slack with the orgs you want to enable. Note: I'll be able to sync this integration to prod only once this PR will be merged. Until then, we will only deploy to staging.

> @FlorentClarret Would it make sense to also add the logs-pipeline and dashboard code within the assets folder for testing alongside the tile in staging or would that need to be done from a separate branch after this is merged?

Hey hey @dayspringjohnson! 

- logs-pipeline: for now, the logs-pipelines still live in our backend. We are in the process of migrating them to the assets in the integration but this is not completed yet. To create and publish the pipeline, you still have to follow the process on the logs-core side. Once the migration will be completed, we will migrate the pipelines for all the existing integrations, nothing to do on your side. So no, do not include any logs-pipeline in this PR please.
- Dashboard: That would make sense to include the dashboard in this PR but that's completely optional. If you want to just create the tile in this PR and open another PR once merged to add the dashboard that's completely fine, so it's up to you 🙂